docs/optimization_guide/nncf/filter_pruning.md

```diff
@@ -0,0 +1,168 @@
+
+### 2. Create NNCF configuration
+Here, you should define NNCF configuration which consists of model-related parameters (`"input_info"` section) and parameters of optimization methods (`"compression"` section). 
+
+@sphinxtabset
+
+@sphinxtab{PyTorch}
+
+@snippet docs/optimization_guide/nncf/code/pruning_torch.py nncf_congig
+
+@endsphinxtab
+
+@sphinxtab{TensorFlow 2}
+
+@snippet docs/optimization_guide/nncf/code/pruning_tf.py nncf_congig
+
+@endsphinxtab
+
+@endsphinxtabset
+
+Here is a brief description of the required parameters of the Filter Pruning method. For full description refer to the [GitHub](https://github.com/openvinotoolkit/nncf/blob/develop/docs/compression_algorithms/Pruning.md) page.
+- `pruning_init` - initial pruning rate target. For example, value `0.1` means that at the begging of training, convolutions that can be pruned will have 10% of their filters set to zero.
+- `pruning_target` - pruning rate target at the end of the schedule. For example, the value `0.5` means that at the epoch with the number of `num_init_steps + pruning_steps`, convolutions that can be pruned will have 50% of their filters set to zero.
+- `pruning_steps` - the number of epochs during which the pruning rate target is increased from `pruning_init` to `pruning_target` value. We recommend to keep the highest learning rate during this period.
+
+### 3. Apply optimization methods
+In the next step, the original model is wrapped by the NNCF object using the `create_compressed_model()` API using the configuration defined in the previous step. This method returns a so-called compression controller and the wrapped model that can be used the same way as the original model. It is worth noting that optimization methods are applied at this step so that the model undergoes a set of corresponding transformations and can contain additional operations required for the optimization. 
+@sphinxtabset
+
+@sphinxtab{PyTorch}
+
+@snippet docs/optimization_guide/nncf/code/pruning_torch.py wrap_model
+
+@endsphinxtab
+
+@sphinxtab{TensorFlow 2}
+
+@snippet docs/optimization_guide/nncf/code/pruning_tf.py wrap_model
+
+@endsphinxtab
+
+@endsphinxtabset
+
+### 4. Fine-tune the model
+This step assumes that you will apply fine-tuning to the model the same way as it is done for the baseline model. In the case of Filter Pruning method we recommend using the training schedule and learning rate similar to what was used for the training of original model.
+
+@sphinxtabset
+
+@sphinxtab{PyTorch}
+
+@snippet docs/optimization_guide/nncf/code/pruning_torch.py tune_model
+
+@endsphinxtab
+
+@sphinxtab{TensorFlow 2}
+
+@snippet docs/optimization_guide/nncf/code/pruning_tf.py tune_model
+
+@endsphinxtab
+
+@endsphinxtabset
+
+> **NOTE**: In the case of distributed multi-GPU training (not DataParallel), you should call `compression_ctrl.distributed()` before the fine-tuning that will inform optimization methods to do some adjustments to function in the distributed mode.
+
+@sphinxtabset
```

Comment:
### 6. Multi-GPU distributed training?

Reply:
I think the note is ok since it's really one additional line of code from the NNCF perspective. Or it's not?

Reply:
applied